src/cleanup_whitespace.py

```diff
@@ -4,36 +4,41 @@
 import sys
 import argparse
 from pathlib import Path
-import codecs
 
 
 def main():
     parser = argparse.ArgumentParser(
         description='''
-Data providers may use the "--tsv_path" option to strip invisible characters from TSVs.
-The cleaned TSV is printed to STDOUT: Use output redirection to save.'''
+Use the "--tsv_in"/"--tsv_out" options to strip invisible characters from TSVs.
+'''
     )
     mutex = parser.add_mutually_exclusive_group(required=True)
     mutex.add_argument(
-        '--tsv_path',
+        '--tsv_in',
         type=Path,
-        metavar='PATH',
+        metavar='INPUT',
         help='TSV to strip padding whitespace from')
     mutex.add_argument(
         '--encoding_test',
         type=str,
         metavar='ENCODING',
         help='Generate test TSV using this encoding')
+    parser.add_argument(
+        '--tsv_out',
+        type=Path,
+        metavar='OUTPUT',
+        help='Destination for clean TSV',
+        required=True)
```

Comment:
yeah, it does break backward compatibility... but since this is just a helper script, I don't think there is much downstream to break. 

This is the crux of the issue:
``` diff
-    writer = csv.writer(sys.stdout, dialect=dialect)
+    writer = csv.writer(output_path.open(mode='w', newline=''), dialect=dialect)
```
Nothing is impossible, but it's not obvious to me how to get `stdout` with `newline=''`.

Reply:
(Interpreting comment as ok-to-merge-but-have-a-question.)